custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -337,17 +345,123 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
         }
     };
 
+    // mobile helpers
+    vm.showSortPopup = false;
+    vm.tooltipDisplayed = -1; // '-1' when none of the tooltips are displayed in sort popup
+    vm.requestDataStartingFrom = 0; // could be any multiple of 10
+    vm.dataSortingDirection = 0; // '0' -> asc, '1' -> desc
+    vm.sortingColumn = 0;
```

Comment:
not sure if worth it but a lot of these could be made constants to make the code more readable. also i think a comment explaining that they map to the datatables API params might be helpful